susemanager/src/mgr_sync/product.py

```diff
@@ -39,6 +40,21 @@ def __init__(self, data):
     def __repr__(self):
         return self.to_ascii_row()
 
+    def __eq__(self, other):
+        return (self.friendly_name == other.friendly_name)
+
+    def __ne__(self, other):
+        return not (self == other)
```

Comment:
As far as I know this is redundant, the default for `__ne__` is already to take the inverse of `__eq__` ([doc](https://docs.python.org/3/reference/datamodel.html#object.__ne__))

Reply:
I copied an example and they also said this, but implemented it anyway :-) I will try to remove it

Reply:
removed